test/integration/scheduler/preemption_test.go

```diff
@@ -49,6 +50,7 @@ var _ = ginkgo.Describe("Preemption", func() {
 				GenerateName: "preemption-",
 			},
 		}
+		_ = features.SetEnable(features.LendingLimit, true)
```

Comment:
Now all tests are running with the feature enabled. That's not the intention.
Put it inside the new `Context` and make sure to set it back to false in `AfterEach`.